### PR TITLE
Fix 2 instances of a typo in the desktop-lite NOTES and README

### PR DESCRIPTION
**Pull request by remlapmot**
This simply amends 'is installed is', which occurs in 2 places in the desktop-lite docs, to 'installed is'.

Alternatively, the amendment could be to 'that is installed is'.

### CODE REVIEW

**Comment by remlapmot** [comment]
@microsoft-github-policy-service agree

File: src/desktop-lite/README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ You can also connect to the desktop using a [VNC viewer](https://www.realvnc.com
 
 ## Customizing Fluxbox
 
-The window manager is installed is [Fluxbox](http://fluxbox.org/). **Right-click** to see the application menu. In addition, any UI-based commands you execute inside the dev container will automatically appear on the desktop.
+The window manager installed is [Fluxbox](http://fluxbox.org/). **Right-click** to see the application menu. In addition, any UI-based commands you execute inside the dev container will automatically appear on the desktop.
```
Comment (samruddhikhandale) [inline src/desktop-lite/README.md:51]: Once this PR is merged, https://github.com/devcontainers/action will run automatically ; which opens a PR that updates the README file based on `devcontainer-feature.json` and `NOTES.md`.

Hence, can we revert the changes to this file to avoid merge conflicts? Thanks!

**Review by samruddhikhandale — DISMISSED** [review]
Nice catch, thanks! ✨ 
Left one minor comment.

File: src/desktop-lite/README.md (hunk 1 above)
Comment (remlapmot) [reply src/desktop-lite/README.md:51]: Thanks, sorry I didn't realise that.

I have made the change on my branch such that only NOTES.md has changes and force pushed, but this PR doesn't seem to have automatically updated as I'm used to seeing - I'm not sure why that is.

Comment (samruddhikhandale) [inline src/desktop-lite/README.md:51]:
```suggestion
The window manager is installed is [Fluxbox](http://fluxbox.org/). **Right-click** to see the application menu. In addition, any UI-based commands you execute inside the dev container will automatically appear on the desktop.
```

Comment (samruddhikhandale) [reply src/desktop-lite/README.md:51]: Hmm, that's super strange! In another comment, I have added a suggestion (which should revert the change), does applying that help?

Comment (remlapmot) [reply src/desktop-lite/README.md]: Thanks - I committed the suggestion - apologies for the extra work.

Comment (samruddhikhandale) [reply src/desktop-lite/README.md]: No worries, thanks!

**Review by samruddhikhandale — APPROVED** [review]
